### PR TITLE
Content card button title

### PR DESCRIPTION
**Pull request by andylushman**
Now we can add custom `button_title` for the `content_cards` on the following pages...
*About
*Program
*Get Involved

### CODE REVIEW

File: app/views/site/about.html.slim [hunk 1]
```diff
@@ -5,4 +5,4 @@ div.About-list-wrapper
   = render 'components/content_card_list', attributes: { 'aria-label': "About page" }  do
     - about_static_data[:items].each do |card|
       = render 'components/content_card_list_item' do
-        = render 'components/content_card', card: card, title: "#{card[:title]}", page_url: "about/#{card[:button_page_url]}"
+        = render 'components/content_card', card: card, button_title: "#{card[:button_title]}", title: "#{card[:title]}", page_url: "about/#{card[:button_page_url]}"
```
Comment (scpaye506) [inline app/views/site/about.html.slim]: do we need to pass in the button_title as its own local here?

Comment (andylushman) [reply app/views/site/about.html.slim]: good call! Just removed it from here and and now using it in `_content_card.html.slim`

File: app/views/site/program.html.slim [hunk 2]
```diff
@@ -5,7 +5,7 @@ div.Program-list-wrapper
   = render 'components/content_card_list', attributes: { 'aria-label': "Programs" }  do
     - programs_static_data[:programs].each do |card|
       = render 'components/content_card_list_item' do
-        = render 'components/content_card', card: card, title: "Program", page_url: "program/"
+        = render 'components/content_card', card: card, button_title: "#{card[:button_title]}", title: "Program", page_url: "program/"
```
Comment (scpaye506) [inline app/views/site/program.html.slim]: and here

Comment (andylushman) [reply app/views/site/program.html.slim]: same here.

**Pull request merged**